frontend/src/components/Layout/index.tsx

```diff
@@ -11,14 +11,15 @@ import LeftMenu from './LeftMenu'
 import { IS_STANDALONE } from 'const/Mode'
 import Loading from 'components/common/Loading'
 import { APP_BAR_HEIGHT } from 'const/Layout'
+import { AppDispatch } from "../../store/store";
 
 const authRequiredPathRegex = /^\/console\/?.*/
 
 const Layout = ({ children }: { children?: ReactNode }) => {
   const user = useSelector(selectCurrentUser)
   const location = useLocation()
   const navigate = useNavigate()
-  const dispatch = useDispatch()
+  const dispatch = useDispatch<AppDispatch>()
```

Comment:
could you explain the reason this definition changed to be required?
It seems that some useDispatch were not added this definition.
What is the differenece?

Reply:
@ReiHashimoto add AppDispatch to be able to use await dispatch

Reply:
@sanglevinh 
Understood, thank you.